dataplane/dataplane.cpp

```diff
@@ -1343,8 +1318,47 @@ void cDataPlane::start()
 
 	bus.run();
 
-	/// run forwarding plane and control plane
-	rte_eal_mp_remote_launch(lcoreThread, this, CALL_MAIN);
+	/// run forwarding plane
+	for (auto& [core, worker] : workers)
+	{
+		if (coreFunctions_.find(core) != coreFunctions_.end())
+		{
+			YANET_LOG_ERROR("Worker: Multiple workloads assigned to core %d\n", core);
+		}
+		YANET_LOG_INFO("Worker assigned to core %d\n", core);
+		coreFunctions_.emplace(core, [worker]() {
+			worker->start();
+		});
```

Comment:
Those are the changes for the separate commit. This is certainly not about "removing Removed runBarrier synchronization"